indra/llcommon/llerror.h

```diff
@@ -95,6 +95,11 @@ const int LL_ERR_NOERR = 0;
 #define LL_STATIC_ASSERT(func, msg) static_assert(func, msg)
 #define LL_BAD_TEMPLATE_INSTANTIATION(type, msg) static_assert(false, msg)
 #else
+#if LL_LINUX
+// We need to include Singal Linden to make the build moat better (also so we ggot access to raise and SIGSEGV
```

Comment:
I appreciate the reference --let's keep the comments germane, however. Remove my name and I will gladly merge. :)

Reply:
Signal Linden has left the building